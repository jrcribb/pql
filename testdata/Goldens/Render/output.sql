WITH "__subquery0" AS (SELECT "State" AS "State", "EventType" AS "EventType", "DamageProperty" AS "DamageProperty" FROM "StormEvents"),
     "__subquery1" AS (SELECT *, 1 AS "foo" FROM "__subquery0"),
     "__subquery2" AS (SELECT "State" AS "State", sum("DamageProperty") AS "TotalDamage" FROM "__subquery1" GROUP BY "State"),
     "__subquery3" AS (SELECT * FROM "__subquery2" ORDER BY "TotalDamage" DESC NULLS LAST LIMIT 10)
SELECT *,
    'barchart' as "render_type",
    'Property Damage by State' as "render_prop_title",
    'State' as "render_prop_xtitle",
    'Total Damage ($)' as "render_prop_ytitle"
FROM "__subquery3";
