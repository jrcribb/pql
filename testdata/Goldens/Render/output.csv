State,TotalDamage,render_type,render_prop_title,render_prop_xtitle,render_prop_ytitle
FLORIDA,6200000,barchart,Property Damage by State,State,Total Damage ($)
MISSISSIPPI,20000,barchart,Property Damage by State,State,Total Damage ($)
GEORGIA,2000,barchart,Property Damage by State,State,Total Damage ($)
ATLANTIC SOUTH,0,barchart,Property Damage by State,State,Total Damage ($)
